t/036-export-fixes-output.t.cpp

```diff
@@ -0,0 +1,109 @@
+#include "clangmetatool-testconfig.h"
+
+#include <memory>
+#include <string>
+#include <unordered_set>
+
+#include <gtest/gtest.h>
+
+#include <clangmetatool/meta_tool_factory.h>
+#include <clangmetatool/meta_tool.h>
+#include <clangmetatool/collectors/find_functions.h>
+
+#include <clang/Frontend/FrontendAction.h>
+#include <clang/Tooling/Core/Replacement.h>
+#include <clang/Tooling/CommonOptionsParser.h>
+#include <clang/Tooling/Tooling.h>
+#include <clang/Tooling/Refactoring.h>
+#include <llvm/Support/CommandLine.h>
+#include <llvm/Support/raw_ostream.h>
+
+#include <iostream>
+
+std::string yaml =
+"---\n"
+"MainSourceFile:  \'\'\n"
+"Replacements:\n"
+"  - FilePath:        \'/home/bgroh2/bbgh/clangmetatool/t/data/036-export-fixes-output/functions.cpp\'\n"
```

Comment:
This has your home dir hardcoded. Though maybe better to load the YAML and validate the contents anyways rather than relying on a specific formatting in it